customerbalancetransaction.go

```diff
@@ -0,0 +1,86 @@
+const (
+	CustomerBalanceTransactionTypeAUABN                    CustomerBalanceTransactionType = "au_abn"
+	CustomerBalanceTransactionTypeEUVAT                    CustomerBalanceTransactionType = "eu_vat"
+	CustomerBalanceTransactionTypeNZGST                    CustomerBalanceTransactionType = "nz_gst"
+	CustomerBalanceTransactionTypeUnknown                  CustomerBalanceTransactionType = "unknown"
+	CustomerBalanceTransactionTypeAdjustment               CustomerBalanceTransactionType = "adjustment"
+	CustomerBalanceTransactionTypeAppliedToInvoice         CustomerBalanceTransactionType = "applied_to_invoice"
+	CustomerBalanceTransactionTypeCreditNote               CustomerBalanceTransactionType = "credit_note"
+	CustomerBalanceTransactionTypeInitial                  CustomerBalanceTransactionType = "initial"
+	CustomerBalanceTransactionTypeInvoiceTooLarge          CustomerBalanceTransactionType = "invoice_too_large"
+	CustomerBalanceTransactionTypeInvoiceTooSmall          CustomerBalanceTransactionType = "invoice_too_small"
+	CustomerBalanceTransactionTypeMigration                CustomerBalanceTransactionType = "migration"
+	CustomerBalanceTransactionTypeReceiverSourceMispayment CustomerBalanceTransactionType = "receiver_source_mispayment"
+	CustomerBalanceTransactionReversal                     CustomerBalanceTransactionType = "reversal"
+)
+
+// CustomerBalanceTransactionParams is the set of parameters that can be used when creating a
+// customer balance transactions.
+// For more details see https://stripe.com/docs/api/customers/create_customer_balance_transaction
+type CustomerBalanceTransactionParams struct {
+	Params      `form:"*"`
+	Amount      *int64  `form:"amount"`
+	Customer    *string `form:"-"`
+	Currency    *string `form:"currency"`
+	Description *string `form:"description"`
+}
+
+// CustomerBalanceTransactionListParams is the set of parameters that can be used when listing
+// customer balance transactions.
+// For more detail see https://stripe.com/docs/api/customers/customer_balance_transactions
+type CustomerBalanceTransactionListParams struct {
+	ListParams `form:"*"`
+	Customer   *string `form:"-"`
+}
+
+// CustomerBalanceTransaction is the resource representing a customer balance transaction.
+// For more details see https://stripe.com/docs/api/customers/customer_balance_transaction_object
+type CustomerBalanceTransaction struct {
+	Amount        int64                          `json:"amount"`
+	Created       int64                          `json:"created"`
+	Currency      Currency                       `json:"currency"`
+	Customer      *Customer                      `json:"customer"`
+	Description   string                         `json:"description"`
+	EndingBalance int64                          `json:"ending_balance"`
+	ID            string                         `json:"id"`
+	Invoice       *Invoice                       `json:"invoice"`
+	Livemode      bool                           `json:"livemode"`
```

Comment:
I think `livemode` field shouldn't be present here. Both api refs, and OpenAPI doesn't have it

Reply:
Yep I put it because I think they made a mistake, all objects should have this field, and I've asked them to fix it already. Will report back!